src/text_cleaning/text_feature_extractor.py

```diff
@@ -0,0 +1,32 @@
+from functools import reduce
+from typing import Dict, List
+
+import pandas as pd
+from tqdm import tqdm
+
+from src.text_cleaning.spell_checker import SmartSpellChecker
+from src.text_cleaning.utils import (count_how_many_words_are_repeating,
+                                     count_misspelled_words, count_punctuation,
+                                     count_words, preprocess_test)
+
+
+class HandcraftedTextFeatureExtractor:
```

Comment:
I believe that class should be outside the `text_cleaning/` folder, somewhere near bert featurizer.

Reply:
Yep, I have created directory feature_extractor and placed there